libraries/Microsoft.Bot.Builder.Core/Adapters/TestAdapter.cs

```diff
@@ -73,17 +73,18 @@ public new TestAdapter Use(IMiddleware middleware)
 
         public async override Task<ResourceResponse[]> SendActivities(ITurnContext context, Activity[] activities)
         {
-            List<ResourceResponse> responses = new List<ResourceResponse>();
+            var responses = new ResourceResponse[activities.Length];
```

Comment:
same comment. Null-check the activities array passed in.

Reply:
✔️